openpgp/packet/private_key.go

```diff
@@ -423,7 +460,6 @@ func (pk *PrivateKey) Encrypt(passphrase []byte) error {
 	}
 
 	pk.s2kParams, err = s2k.Generate(rand.Reader, s2kConfig)
-	privateKeyBytes := privateKeyBuf.Bytes()
```

Comment:
Also here I prefer the old code of only calling `privateKeyBuf.Bytes()` once, instead of multiple times